src/index.js

```diff
@@ -160,3 +168,18 @@ export function register(ReactComponent, tagName, mapping = {}) {
 
   return customElements.define(tagName, WebReactComponent);
 }
+
+// register is default export
+export default register;
+
+// additionally everything is exported as named export for convenience
+export {
+  register,
```

Comment:
I think the `convenience` here is more confusing than that it helps. I would keep it simple here just register as the default exports and Types as a named one. The other ones (string etc) can be reached via Types. I would leave the importing to the user, it's not that hard.